recipes/nest-desktop/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "nest-desktop" %}
+{% set version = "3.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nest-desktop-{{ version }}.tar.gz
+  sha256: 49051aeefc8be189ae7dfb64ad983b7a0a5636765a79febd3347194b57ba61ba
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
```

```suggestion
  skip: True  # [not unix]
```
On second review, it looks like your recipe does not qualify for `noarch: python` because you are shipping a bash script (which doesn't work on Windows).

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

---

First, to clarify our concept: we have a python package `nest_desktop` and a bash script `nest-desktop`.
The user can start the server instance of NEST Desktop in both ways.

NEST Desktop uses Python module `http.server` module which runs on `from nest_desktop import app; app.run()`.

The bash script to start NEST Desktop is only a wrapper to Python module, we could ignore it for `noarch`.
```python3 -m nest_desktop.app ${HOST} ${PORT}```